src/components/pages/home.js

```diff
@@ -9,9 +11,14 @@ import handHeart from "../../assets/img/hand-heart.svg";
 import lockHeart from "../../assets/img/lock-heart.svg";
 
 class Home extends React.Component {
+
   render() {
+
     return (
       <section id="home">
+        <LandingNav />
+        <div id="hero">
```

Comment:
@tombjarne this currently shows as an empty orange bar, is that right ?

Reply:
Yes, it is a placeholder for the content to come.